web/src/routes/CategoriesPage.tsx

```diff
@@ -120,12 +120,15 @@ const CategoriesPage = ({ city, pathname, cityCode, languageCode }: CityRoutePro
     }
   })
 
+  const pageTitle = `${!category?.isRoot() ? `${category?.title} - ` : ''}${city.name}`
```

Comment:
Should we change this to
```suggestion
  const pageTitle = `${category && !category.isRoot() ? category.title : t('dashboard:localInformation')} - ${city.name}`
```